tests/scripts/pulp_rpm/test_content.sh

```diff
@@ -140,6 +140,11 @@ do
   fi
 done
 
+if pulp debug has-plugin --name "rpm" --specifier ">=3.20.0"
```

Comment:
Anecdotally, I must claim I am quite pleased to see how readable these conditions have become over time.